integrationTests/ts/__tests__/suites.ts

```diff
@@ -41,13 +41,7 @@ const executeSuite = async (data: any, expect: any) => {
         const config = loadYaml()
         const coordinatorKeypair = new Keypair()
 
-        const maciState = new MaciState(
-            coordinatorKeypair,
-            config.constants.maci.stateTreeDepth,
-            config.constants.maci.messageTreeDepth,
-            config.constants.maci.voteOptionTreeDepth,
-            config.constants.maci.maxVoteOptions,
-        )
+        const maciState = new MaciState()
```

Comment:
Why no longer passing those arguments?

Reply:
Because now that there are type declarations, TS complaints that we do not need them, if you look at the constructor of MaciState it does not take any.